packages/server/src/modules/teams/services/index.ts

```diff
@@ -1,23 +1,35 @@
 import { Team } from "@prisma/client";
+import { range } from "lodash";
 import { database } from "../../../database";
 
 const model = database.team;
 type Model = Team;
 
 export { services };
 
-const crudServices = { create, getMany, get, update };
+const crudServices = { create, getMany, get, update, createMany };
 
 const services = { ...crudServices };
 
 async function create(
-  document: Omit<Model, "id" | "scenarioName">
+  document: Omit<Model, "id" | "scenarioName" | "showInGame">
 ): Promise<Model> {
   return model.create({ data: document });
 }
 
+async function createMany(gameId: number, quantity: number) {
+  const teams = (await model.findMany({ where: { gameId } })).filter(
+    (team: Model) => team.name !== "Aucune équipe"
+  );
+  for await (const index of range(1, quantity + 1)) {
```

Comment:
@Baboo7 @louis-sanna-eki Ici le linter me dit 
`error  iterators/generators require regenerator-runtime, which is too heavyweight for this guide to allow them. Separately, loops should be avoided in favor of array iterations` 
Sauf que si je passe par une map le problème c'est que la création se fait pas dans l'ordre et on a des cas où on a une équipe 8 avant 6 et 7 par exemple, sauf qu'on trie par id pour l'affichage donc pas ouf. 
Est-ce que ça vous choque de laisser ça et ignorer le linter ? Sachant que c'est de la création d'équipes où on va faire max 5 ou 10 calls ça devrait pas être trop lourd à chaque fois

Reply:
@remiriv oui pas de problème pour mettre un ts-ignore sur cette ligne. Et je crois qu'on a pas besoin du `await` dans `for await`

Reply:
`/Users/remiriviere/Projects/perso/data-for-good/offseason_ogre/packages/server/src/modules/teams/services/index.ts
  27:5  error  Unexpected `await` inside a loop  no-await-in-loop`
Si j'enlève le await de `for await` le linter me pète une erreur sur la ligne d'en dessous mais pas d'erreur si je le laisse, étrange